examples/nlp/text_classification_tf_keras/model_def.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+from determined.keras import TFKerasTrial, TFKerasTrialContext, InputData
+import data
+
+
+class MultiTextClassificationTrial(TFKerasTrial):
+    def __init__(self, context: TFKerasTrialContext) -> None:
+        self.context = context
```

Comment:
Have you tested dtrain here? You may need to add something like https://github.com/determined-ai/determined/blob/master/examples/tutorials/mnist_pytorch/model_def.py#L32. This is to ensure that for each rank/GPU data is downloaded to a separate directory so there are no cache conflicts? Otherwise, things may error out on some runs depending on what your GPU has cached in memory.

Reply:
good catch. added rank as a parameter for download directory